deployment-aws/puppet/cromwell/manifests/config.pp

```diff
@@ -19,6 +19,13 @@
     group   => $cromwell::user,
   }
 
+  file { '/etc/cromwell_cloudwatch_config.json':
```

Comment:
"If you create or edit the CloudWatch agent configuration file manually, you can give it any name. For simplicity in troubleshooting, we recommend that you name it /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json on a Linux server "
https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html#Saving-Agent-Configuration-File

This is probably a bit of an overkill in this case, so maybe shorten it to `/opt/amazon-cloudwatch-agent.json`?

Reply:
I did create a customized file using their wizard (which is the contents of `cromwell_cloudwatch_config.json.epp`) and I can specify any path when starting the agent. Since we were already putting other templated-files into `/etc`, I had just kept that "convention"